boot/cmdline_test.go

```diff
@@ -206,8 +206,8 @@ func (s *kernelCommandLineSuite) TestComposeCommandLineManagedHappy(c *C) {
 
 	cmdline, err := boot.ComposeRecoveryCommandLine(model, "20200314")
 	c.Assert(err, IsNil)
-	c.Assert(cmdline, Equals, "panic=-1 snapd_recovery_mode=recover snapd_recovery_system=20200314")
+	c.Assert(cmdline, Equals, "snapd_recovery_mode=recover snapd_recovery_system=20200314 panic=-1")
```

Comment:
these tests didn't reflect the real command line right?

if I understand in the end we will produce the same command line that in the grub cfgs currently?

Reply:
@bboozzoo confirmed that in the end the commad lines are unchanged.